pricing.html

```diff
@@ -0,0 +1,268 @@
+
+        <div class="nav__logo">
+            <img src="img/Vector.png" alt="">
+        </div>
+        <div class=" nav__menu ">
+            <div class="nav__list">
+                <a href="home.html" class="nav__item">Home</a>
+                <a href="product.html" class="nav__item">Product</a>
+                <a href="pricing.html" class="nav__item">Pricing</a>
+                <a href="#" class="nav__item">About Us</a>
+                <a href="#" class="nav__item">Blog</a>
+                <a href="#" class="nav__item">Contact</a>
+
+
+            </div>
+            <a href="#" class="button ">Free Trial</a>
+        </div>
+
+    </nav>
+    <div class="hero-content ">
+        <h1 class="title title--pricing ">
+            Pricing plans that suit you
+        </h1>
+        <p class=" subtitle subtitle--pricing  ">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
+        </p>
+        <div class="hero-content__input hero-content__input--pricing">
+
+            <a href="#" class="button button--free-trial">Monthly</a>
+            <p class="hero-content__yearly">Yearly</p>
+
+
+        </div>
+        <div class="hero-content__plans">
+
+            <div class="plan-card ">
+                <div class="plan-card__title  ">
+                    <img src="img/plan-regular.svg" alt="" class="plan-card__icon">
+                    <div class="plan-card__profile ">
+                        <h1 class="plan-card__name">Regular </h1>
+                        <p class="plan-card__surname">Starter Plan</p>
+
+                    </div>
+                </div>
+                <ul class="plan-card__list ">
+
+                    <li class="plan-card__item "><img src="img/plan-iconlist.svg" alt="" class="plan-card__list-icon"> <span>Limited Projects</span></li>
+                    <li class="plan-card__item "><img src="img/plan-iconlist.svg" alt="" class="plan-card__list-icon"> <span>Regular Support Business</span></li>
+                    <li class="plan-card__item "><img src="img/plan-iconlist.svg" alt="" class="plan-card__list-icon"> <span>1 month Free Trial</span></li>
+                    <li class="plan-card__item "><img src="img/plan-iconlist.svg" alt="" class="plan-card__list-icon"> <span>3GB storage</span></li>
+                    <li class="plan-card__item "><img src="img/plan-iconlist.svg" alt="" class="plan-card__list-icon"> <span>Ads Preview</span></li>
+
+                </ul>
+                <p class="plan-card__subtitle  ">Lorem ipsum dolor sit amet, consetetur sadipscing elitr.</p>
+                <div class="plan-card__pricing ">
+                    <h1 class="plan-card__price">Free</h1>
+                    <p class="plan-card__subtitle ">For Limited Period</p>
+                    <a href="" class="button">Get Started</a>
+
+                </div>
+
+            </div>
+            <!-- card two -->
+            <div class="plan-card plan-card--platinum ">
```

Comment:
_**--platinum**_ shouldn't be a modifier.. maybe use like.... plan-card--outlined or plan-card--active